index.md

```diff
@@ -11,11 +11,12 @@ image:
   <p>For more information see <a href="{{ site.github.url }}/about/overview">Overview of Social Feed Manager</a> and the <a href="https://sfm.readthedocs.org">documentation</a>.</p> 
```

Comment:
Suggest "collect, manage, and export datasets" instead of "define and create collections". Also, "innovative and impactful." Also, put "Members of the GW ..." in separate paragraph.